client/packages/common/src/plugins/components/PluginLoader.tsx

```diff
@@ -50,9 +39,8 @@ export const PluginLoader = ({
     );
   };
 
-  const url = `${Environment.PLUGIN_URL}/${name}${Environment.PLUGIN_EXTENSION}`;
   const Component = React.lazy<ComponentType<{ data: unknown }>>(
-    loadPlugin({ plugin: name, url, module, scope })
+    loadPluginModule({ plugin: name, module, scope })
```

Comment:
`module` here is the name of the es6 module exposed by webpack module federation (wmf)